internal/pkg/mining/worker_test.go

```diff
@@ -53,33 +53,24 @@ func TestLookbackElection(t *testing.T) {
 	for i := 1; i < int(lookback); i++ {
 		head = builder.AppendOn(head, 1)
 	}
-	ancestors := builder.RequireTipSets(head.Key(), int(lookback))
 
 	st, pool, addrs, bs := sharedSetup(t, mockSignerVal)
 	getStateTree := func(c context.Context, tsKey block.TipSetKey) (state.Tree, error) {
 		return st, nil
 	}
-	getAncestors := func(ctx context.Context, ts block.TipSet, newBlockHeight abi.ChainEpoch) ([]block.TipSet, error) {
-		return ancestors, nil
-	}
 
+	rnd := &consensus.FakeChainRandomness{Seed: 0}
 	minerAddr := addrs[3]      // addr4 in sharedSetup
 	minerOwnerAddr := addrs[4] // addr5 in sharedSetup
 
 	messages := chain.NewMessageStore(bs)
 
 	t.Run("Election sees ticket lookback ancestors back", func(t *testing.T) {
-		electionTicket, err := ancestors[lookback-1].MinTicket()
-		require.NoError(t, err)
-		mem := consensus.NewMockElectionMachine(func(ticket block.Ticket) {
```

Comment:
Was the fix skipped for expediency (which is ok with me)?  If not I'm suspicious it should be straightforward to do mock machines in the new model but maybe I'm missing something there.  Since it looks like we plan to come back we should skip this louder as its not testing anything right now.

Reply:
The test is still skipped because it depends on code that was commented-out during the VM integration, but I've restored code that checks this (that may or may not work).